SmartDeviceLink/SDLFileManager.m

```diff
@@ -526,11 +524,28 @@ - (float)sdl_uploadPercentage:(float)totalBytes uploadedBytes:(float)uploadedByt
  *  @return                 True if the file still needs to be (re)sent to Core; false if not.
  */
 - (BOOL)sdl_canFileBeUploadedAgain:(nullable SDLFile *)file maxUploadCount:(UInt8)maxUploadCount failedFileUploadsCount:(NSMutableDictionary<SDLFileName *, NSNumber<SDLUInt> *> *)failedFileUploadsCount {
-    if (!file || [self hasUploadedFile:file]) {
+    if (![self.currentState isEqualToString:SDLFileManagerStateReady]) {
+        SDLLogE(@"File named %@ failed to upload. The file manager has shutdown so the file will not be sent again.", file.name);
```

Comment:
Make this a warning. Change string to `File named %@ failed to upload. The file manager has shutdown so the file upload will not retry.`

Reply:
Fixed